cloudferry_devlab/cloudferry_devlab/tests/base.py

```diff
@@ -418,6 +432,20 @@ def get_abs_path(self, file_path):
         return os.path.join(os.path.dirname(self.results_path), file_path)
 
 
+def get_nosetest_cmd_attribute_val(attribute):
+    env = os.environ
+    manager = DefaultPluginManager()
+    cfg_files = all_config_files()
+    tmp_config = Config(env=env, files=cfg_files, plugins=manager)
+    tmp_config.configure()
+    try:
+        attr_list = getattr(tmp_config.options, 'attr')
+        value = dict(token.split('=') for token in attr_list)
+        return value[attribute]
+    except TypeError:
+        return None
```

Comment:
Dependency on environment, all config files and plugins looks super strange given that all data is generated by single script which is part of the tests. I'd suggest that this part is removed and alternative way should be found to get the result.